src/borg/testsuite/helpers_test.py

```diff
@@ -1408,12 +1408,14 @@ def test_passphrase_new_retries(self, monkeypatch):
     def test_passphrase_repr(self):
         assert "secret" not in repr(Passphrase("secret"))
 
-    def test_passphrase_wrong(self, capsys, monkeypatch):
+    def test_passphrase_wrong_debug_enabled(self, capsys, monkeypatch):
+        monkeypatch.setenv("BORG_DEBUG_PASSPHRASE", "YES")
         monkeypatch.setenv("BORG_PASSPHRASE", "wrong_passphrase")
         monkeypatch.setenv("BORG_PASSCOMMAND", "echo wrong_passphrase")
         monkeypatch.setenv("BORG_PASSPHRASE_FD", "123")
 
         with pytest.raises(PassphraseWrong) as exc_info:
+            Passphrase.display_debug_info("wrong_passphrase")
             raise PassphraseWrong("wrong_passphrase")
```

Comment:
guess you just had removed the passphrase argument.

Reply:
I accidently overlooked that, thanks for pointing it out!